google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonToProtoMessage.java

```diff
@@ -93,14 +93,11 @@ public class JsonToProtoMessage implements ToProtoConverter<Object> {
           .appendLiteral(' ')
           .optionalEnd()
           .optionalStart()
-          .appendOffset("+HH:MM", "+00:00")
+          .appendZoneOrOffsetId()
           .optionalEnd()
           .optionalStart()
           .appendZoneText(TextStyle.SHORT)
           .optionalEnd()
-          .optionalStart()
-          .appendLiteral('Z')
-          .optionalEnd()
           .toFormatter()
           .withZone(ZoneOffset.UTC);
```

Comment:
This fix works for the current tests. Note that the literal `Z` is dropped from the config because `appendZoneOrOffsetId()` uses `Z` as the no-zone string

Reply:
Added a test for `+00:00` in https://github.com/googleapis/java-bigquerystorage/pull/2780/commits/260c835c175dae427b9d518818511a288ec7ad21